src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -141,10 +142,21 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
                     (from, nick, role, hidden, statsid, status, identity) => {
                         console.log("Jibri got MUC_MEMBER_JOINED: ", from, identity);
                         if (!hidden && identity) {
-                            window._jibriParticipants.push(identity);
+                            identity.lastKnownDisplayName = nick;
+                            window._jibriParticipants.set(from, identity);
                         }
                     }
                 );
+                APP.conference._room.room.addListener(
+                    "xmpp.display_name_changed",
+                    (jid, displayName) => {
+                        const identity = window._jibriParticipants.get(jid);
+                        if (identity) {
+                            identity.lastKnownDisplayName = displayName;
```

Comment:
Pl set the display name property instead, so the JaaS webhook payload doesn't change since it takes it from the metadata file.

Reply:
So identity already has the display name that is coming from the jwt. Do you want to override that one and lost any values from it?

Reply:
I see, to overwrite name in user:
```
{
  "user": {
    "id": "123",
    "name": "damencho",
    "avatar": "https://"
  },
  "group": "456"
}
```